src/dist/replication/lib/replica.h

```diff
@@ -494,6 +494,9 @@ class replica : public serverlet<replica>, public ref_counter, public replica_ba
     // ballot when starting partition split and split will stop if ballot changed
     // _child_init_ballot = 0 if partition not in partition split
     ballot _child_init_ballot{0};
+    // in normal case _partition_version = partition_count-1
```

Comment:
Write comments to public members rather than to private members.

Reply:
Got it~